app/views/collections/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<% title_string = "#{@user.name}'s Series" %>
+
+<%= content_for :page_meta do %>
+  <% title title_string %>
+  <link rel="canonical" href="<%= app_url("/about") %>" />
+  <meta name="description" content="<%= title_string %> on <%= community_name %>">
+  <meta name="keywords" content="<%= SiteConfig.meta_keywords[:default] %>">
+
+  <meta property="og:type" content="article" />
+  <meta property="og:url" content="<%= app_url("/about") %>" />
+  <meta property="og:title" content="About <%= community_name %>" />
+  <meta property="og:image" content="<%= SiteConfig.main_social_image %>" />
+  <meta property="og:description" content="<%= community_qualified_name %> is great!" />
+  <meta property="og:site_name" content="<%= community_qualified_name %>" />
+
+  <meta name="twitter:card" content="summary_large_image">
+  <meta name="twitter:site" content="@<%= SiteConfig.social_media_handles["twitter"] %>">
+  <meta name="twitter:title" content="About <%= community_name %>">
+  <meta name="twitter:description" content="<%= community_qualified_name %> is great!">
+  <meta name="twitter:image:src" content="<%= SiteConfig.main_social_image %>">
+<% end %>
```

Comment:
could this be put in a partial for both index and show pages?

Reply:
Definitely!